x-pack/plugins/security_solution/public/explore/network/containers/network_top_n_flow/index.tsx

```diff
@@ -160,8 +167,9 @@ export const useNetworkTopNFlow = ({
   useEffect(() => {
     if (!skip && networkTopNFlowRequest) {
       search(networkTopNFlowRequest);
+      searchTotalCount(networkTopNFlowRequest);
     }
-  }, [networkTopNFlowRequest, search, skip]);
+  }, [networkTopNFlowRequest, search, searchTotalCount, skip]);
 
-  return [loading, networkTopNFlowResponse];
+  return [loading || loadingTotalCount, networkTopNFlowResponse];
```

Comment:
NIT:

not sure if you have  a convention for namings, but

`loading` namewise is ok because it's clear that it's a boolean
but `loadingTotalCount` is ambiguous, it also reads as "total count of loadings"

That's why  using a `is<X>`, `has<X>`, `did<X>`, `should<X>` to denote booleans
is a popular naming convention that makes sure booleans are easily highlighted and unambiguous
when reading code.

Wdyt?